tool-plugins/intellij/src/main/java/org/ballerinalang/plugins/idea/completion/BallerinaCompletionUtils.java

```diff
@@ -597,6 +597,9 @@ private static LookupElement createConnectorLookupElement(@NotNull IdentifierPSI
                                                                             insertHandler) {
         List<LookupElement> lookupElements = new LinkedList<>();
         for (IdentifierPSINode connector : connectors) {
+            if (connector == null) {
```

Comment:
we can simplify this IF to 
```
if(connector != null) {
    // do your things
}
```
 so we can reduce a line. This is just a suggestion.